units/units_util.hpp

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (c) 2019-2021,
+Lawrence Livermore National Security, LLC;
+See the top-level NOTICE for additional details. All rights reserved.
+SPDX-License-Identifier: BSD-3-Clause
+*/
```

Comment:
You should add a copyright or at least acknowledgement so we can track the origin of the code here.

Reply:
Since this is a contribution to this project, I think the LLNL copyright should just be kept? `git blame` will always show who contributed individual lines.